packages/app/src/web3/Server.js

```diff
@@ -1,25 +1,24 @@
 import axios from 'axios'
```

Comment:
can't we just use fetch for this stuff?

Reply:
No, we can create an issue to stop using axios

Reply:
So you are using it in other places as well? Why is that?

Reply:
No particular reason, we can simply use fetch. We are using it for the monitoring service as well

Reply:
Created [this issue](https://github.com/aragonone/court-backend/issues/139) to tackle that